src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/layout/Resources/ui/js/Videolink.js

```diff
@@ -0,0 +1,100 @@
+export default class Videolink {
+
+    constructor() {
+        Array.from(document.querySelectorAll('.js-videolink-placeholder')).forEach((placeholder) => {
```

Comment:
`Array.from` is not supported in IE and is one of those Array prototype methods that Babel doesn't catch if you use the es2015 preset. A better solution would be to use `Array.prototype.slice.call(...)`. 
(Or look for a preset that transpiles Array prototype methods)